Documentation/Google-Driva-Ludo.md

```diff
@@ -0,0 +1,43 @@
+# Dokumentation
+
+Dokumentation för projektet finns i den google drive mapp. Där har vi våra funderingar och tankar om hur vi ska gå till väga. Titta gärna igenom det.
+
+
+
+## Hela dokumentationsmappen
+
+https://drive.google.com/drive/folders/1pV2S6IWSh8DLRkNVta5LIvXH1TokzQYa?usp=sharing
+
+
+
+## CRC-kort
+
+https://drive.google.com/drive/folders/1LF5EKn7WgRvlr1l0H4STa2ITIlYGRi40?usp=sharing
+
+
+
+## Mindmaps
+
+https://drive.google.com/drive/folders/1LsxtpD2fNqKd4CQi6DTtqNFptu3nOwGr?usp=sharing
+
+
+
+## Rules of engagement
+
+###### Regler som ska följas och Milstolpar och problem som uppstår som vi ska dokumentera
+
+https://drive.google.com/drive/folders/1uT1EEBtHiO04EGh3z5rloy1_HCxwiwY5?usp=sharing
+
+
+
+## Spelplan
+
+###### Visuell representation av hur ska spelplanen ska se ut
+
+https://drive.google.com/drive/folders/1giE0S0rs1dBZJwMNoZOqD1ILlbqWgrwv?usp=sharing
+
+
+
+## User stories
+
+https://drive.google.com/drive/folders/1Nc-3ViVNKqNsxmh7711QlzJOnYBUfQvh?usp=sharing
```

Comment:
Maybe you should also write descriptions of your userstories, to concretizise methods, functionality etc in your program :)

Reply:
Schysst tanke! Alltså för att mer konkret peka på vad vi behöver göra utifrån en user story?